azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -74,6 +74,11 @@ def __del__(self):
         """
         self.amqp_svc_client.disconnect_sync()
 
+    def __ensure_quoted(self, etag):
+        if not isinstance(etag, str) or (len(etag) > 1 and etag[0] == '"' and etag[-1] == '"'):
+            return etag
+        return '"' + etag + '"'
+
```

Comment:
The service expects etags to be wrapped in double quotes. To avoid breaking users who have figured this out, I only wrap etags that haven't already been wrapped by the user. A [similar fix was made in Node](https://github.com/Azure/azure-iot-sdk-node/pull/613).